builder/parser/line_parsers.go

```diff
@@ -209,34 +208,28 @@ func parseString(rest string) (*Node, map[string]bool, error) {
 
 // parseJSON converts JSON arrays to an AST.
 func parseJSON(rest string) (*Node, map[string]bool, error) {
-	var (
-		myJson   []interface{}
-		next     = &Node{}
-		orignext = next
-		prevnode = next
-	)
-
+	var myJson []interface{}
 	if err := json.Unmarshal([]byte(rest), &myJson); err != nil {
 		return nil, nil, err
 	}
 
+	var top, prev *Node
 	for _, str := range myJson {
 		switch str.(type) {
 		case string:
-		case float64:
-			str = strconv.FormatFloat(str.(float64), 'G', -1, 64)
```

Comment:
Also, we never describe it this way.  Can you find me `Dockerfile`s in the wild that use it legitimately?  Is there actually a case where a `float64` is _preferred_ to a string?  For reals?